src/libostree/ostree-fetcher-util.h

```diff
@@ -35,14 +35,8 @@ static inline gboolean
 _ostree_fetcher_tmpf_from_flags (OstreeFetcherRequestFlags flags, int dfd, GLnxTmpfile *tmpf,
                                  GError **error)
 {
-  if ((flags & OSTREE_FETCHER_REQUEST_LINKABLE) > 0)
```

Comment:
It looks like this is a dead flag with this removal. Since this is internal API (I think), can it be removed?

Reply:
Yeah, I felt it was clearer to keep the flag even if unused just to clarify the simplicity of the change.

May do some other cleanup though after this.